DEVELOPMENT.md

```diff
@@ -86,7 +86,7 @@ This project takes a very collaborative and [agile](https://agilemanifesto.org/)
 
 ## Releases
 
-Releases follow the [CalVer](https://calver.org/) versioning using a `YYY.MM.DD` format. Optionally include a `-N` if more than one releases are published in the same day.
+Releases follow the [CalVer](https://calver.org/) versioning using a `YYYY.MM.DD-N` format. Start with `-1` for a release, incrementing if there is another release on the same day.
```

Comment:
Why a dash and not a dot?

Reply:
@andycochran we discussed this. The dot might be construed as a continuation of the date format, while the dash makes it more clear that it is not part of the date format and incremental.